element_miniscope/miniscope.py

```diff
@@ -217,62 +351,28 @@ def create1_from_processing_task(self, key, is_curated=False, curation_note=''):
+    -> Channel.proj(motion_correct_channel='channel') # channel used for motion correction in this processing task
     """
 
     class RigidMotionCorrection(dj.Part):
-        definition = """ 
+        definition = """
         -> master
         ---
         outlier_frames=null : longblob  # mask with true for frames with outlier shifts (already corrected)
         y_shifts            : longblob  # (pixels) y motion correction shifts
         x_shifts            : longblob  # (pixels) x motion correction shifts
-        z_shifts=null       : longblob  # (pixels) z motion correction shifts (z-drift) 
+        z_shifts=null       : longblob  # (pixels) z motion correction shifts (z-drift)
         y_std               : float     # (pixels) standard deviation of y shifts across all frames
         x_std               : float     # (pixels) standard deviation of x shifts across all frames
         z_std=null          : float     # (pixels) standard deviation of z shifts across all frames
         """
 
-    class NonRigidMotionCorrection(dj.Part):
-        """
-        Piece-wise rigid motion correction
-        - tile the FOV into multiple 3D blocks/patches
-        """
-        definition = """ 
-        -> master
-        ---
-        outlier_frames=null             : longblob      # mask with true for frames with outlier shifts (already corrected)
-        block_height                    : int           # (pixels)
-        block_width                     : int           # (pixels)
-        block_depth                     : int           # (pixels)
-        block_count_y                   : int           # number of blocks tiled in the y direction
-        block_count_x                   : int           # number of blocks tiled in the x direction
-        block_count_z                   : int           # number of blocks tiled in the z direction
-        """
-
-    class Block(dj.Part):
-        definition = """  # FOV-tiled blocks used for non-rigid motion correction
-        -> master.NonRigidMotionCorrection
-        block_id        : int
-        ---
-        block_y         : longblob  # (y_start, y_end) in pixel of this block
-        block_x         : longblob  # (x_start, x_end) in pixel of this block
-        block_z         : longblob  # (z_start, z_end) in pixel of this block
-        y_shifts        : longblob  # (pixels) y motion correction shifts for every frame
-        x_shifts        : longblob  # (pixels) x motion correction shifts for every frame
-        z_shifts=null   : longblob  # (pixels) x motion correction shifts for every frame
-        y_std           : float     # (pixels) standard deviation of y shifts across all frames
-        x_std           : float     # (pixels) standard deviation of x shifts across all frames
-        z_std=null      : float     # (pixels) standard deviation of z shifts across all frames
-        """
```

Comment:
I think some groups use `NoRMCorre` for motion correction so we would need these part tables (possibly without the depth/z attributes).